app/controllers/users_controller.rb

```diff
@@ -66,6 +66,9 @@ def refresh_user_list
       find_or_create_user(user_json)
     end
 
+    users_not_in_json = User.active.in_banana.where.not(uid: users_json.pluck('id'))
```

Comment:
Also check that their credit is 0

Reply:
It is already checked by the `:no_deactivation_when_nonzero_credit` validation, should I explicitly check it here too?